cleo/commands/completions_command.py

```diff
@@ -120,266 +120,148 @@ def handle(self) -> int:
         return 0
 
     def render(self, shell: str) -> str:
-        return getattr(self, f"render_{shell}")()
-
-    def render_bash(self) -> str:
-        template = TEMPLATES["bash"]
+        return getattr(self, f"render_{shell}")()  # type: ignore[no-any-return]
```

Comment:
how about 
```suggestion
        return {
            "bash": render_bash,
            "zsh": render_zsh,
            "fish": render_fish
         }[shell]()
```

Reply:
Don't know however how mypy will like this (if it actually solves the error or not)

Reply:
dropped the dynamicism in favor of simple `if` blocks